src/components/config/validate.ts

```diff
@@ -71,7 +84,7 @@ const PROBE_REQUEST_INVALID_URL = setInvalidResponse(
   'Probe request URL should start with http:// or https://'
 )
 const PROBE_REQUEST_INVALID_METHOD = setInvalidResponse(
-  'Probe request method should be GET or POST only'
+  'Probe request method is invalid!'
```

Comment:
Do we need more specific error message for this one? For example what is current method and what is available methods?

Reply:
Should be fixed in the newest update! :D